src/components/graphs/ChangeOverTimeGraph/ChangeOverTimeGraph.js

```diff
@@ -32,6 +32,28 @@ class ChangeOverTimeGraphDisplay extends React.Component {
   // This component is wrapped with `withAsyncData` to inject the
   // statistics that are fetched asynchronously, according to the
   // selected region and climatological time period.
+  constructor(props) {
+    super(props);
+    this.state = {
+      median: null
+    };
+  }
+
+  componentDidMount() {
```

Comment:
All the other data this component uses seems to be fetched with `withAsyncData`. It's a little confusing to fetch just this one piece here. Is there a particular reason to do ti this way?

In any case, if you decide to keep the data fetch handled this way, edit the comments above to note that this is no longer a state-free component and perhaps note which data comes to the component via which pathways.

Reply:
I agree that we should maintain consistent data fetching using withAsyncData. I have updated the relevant code to ensure the medians are fetched similarly to the summary statistics.